gui/storage/forms.py

```diff
@@ -1765,7 +1765,10 @@ def __init__(self, *args, **kwargs):
         volnames = [
             o.vol_name for o in models.Volume.objects.filter(vol_fstype='ZFS')
         ]
```

Comment:
Ok I see that it has changed since your commit (its no longer there in the master code), nvm me moving on...